sound/soc/sof/intel/hda-dsp.c

```diff
@@ -29,6 +29,12 @@ module_param_named(enable_trace_D0I3_S0, hda_enable_trace_D0I3_S0, bool, 0444);
 MODULE_PARM_DESC(enable_trace_D0I3_S0,
 		 "SOF HDA enable trace when the DSP is in D0I3 in S0");
 #endif
+static bool hda_S0ix_host_dma_stop;
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG)
+module_param_named(S0ix_host_dma_stop, hda_S0ix_host_dma_stop, bool, 0444);
```

Comment:
since this is for debugging only anyway, you could make it writable for root to switch it on and off at run-time?